questionpy_server/worker/pool.py

```diff
@@ -47,6 +62,14 @@ def __init__(self, max_workers: int, max_memory: int, worker_type: type[Worker]
         self.max_workers = max_workers
         self.max_memory = max_memory
 
+        # TODO: Make this configurable (#137)
+        self._limits_per_worker = WorkerResourceLimits(max_memory=200 * MiB, max_cpu_time_seconds_per_call=10)
```

Comment:
Das soll schon in get_worker bleiben. Die Limits sollen nicht fest sein, siehe https://github.com/questionpy-org/questionpy-server/issues/137#issuecomment-2809907297

Reply:
Ich habe diese Limits in eine Konstante gezogen, den Check zurück in `get_worker` (wobei ich denke, dass der dort mittelmäßig viel Sinn macht) und den Kommentar ausführlicher gemacht.